### DISCUSSION TITLE
Is it possible to print QR CODE? If not, do you plan to support QR CODE?

### DISCUSSION THREAD

**Discussion by ARMS-Co-LTD** [BinaryKits/ZPLUtility#167] [Q&A] [answered]
Is it possible to print QR CODE? If not, do you plan to support QR CODE?

**Comment by primo-ppcg** [comment, marked as answer]
QR Codes are already supported with the `^BQ` command ([zpl documentation](https://docs.zebra.com/content/tcm/us/en/printers/software/zebra-zpl-ii,-zbi-2,-set-get-do,-mirror,-wml-programming-guide/c-zpl-zpl-commands/r-zpl-bq.html)):

```zpl
^XA
^BQN,2,8
^FDMA,BinaryKits.Zpl.Viewer^FS
^XZ
```
